openml/runs/trace.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+class OpenMLRunTrace(object):
+    """OpenML Run Trace: parsed output from Run Trace call
+
+    Parameters
+    ----------
+    FIXME
+
+    """
+
+    def __init__(self, run_id, trace_iterations):
+        self.run_id = run_id
+        self.trace_iterations = trace_iterations
+
+    def get_selected_iteration(self, fold, repeat):
+        for (r, f, i) in self.trace_iterations:
+            if r == repeat and f == fold and self.trace_iterations[(r, f, i)].selected == True:
+                return i
+        raise ValueError('Could not find the selected iteration for rep/fold %d/%d' %(repeat,fold))
+
+    def __str__(self):
+        return '[Run id: %d, %d trace iterations]' %(self.run_id, len(self.trace_iterations))
```

Comment:
Added a comment that this will change. Was actually very useful for debugging, seems suboptimal to remove